airflow/www/templates/airflow/trigger_dag.html

```diff
@@ -27,11 +27,22 @@
     <form action="{{ url_for('airflow.trigger_with_conf') }}" method="post">
       <input type="hidden" value="{{ dag.dag_id }}" name="dag_id">
       <input name=_csrf_token type=hidden value="{{ csrf_token() }}">
-      <h4 class="pull-right">
-          <a href="#" class="btn btn-info" onclick="$(this).closest('form').submit()">Go!</a>
-      </h4>
-      <br>
       <br>
+      <table class="pull-right">
```

Comment:
Add new input box next to 'GO!' button to customise the Dag run id. So when multiple people running multiple dag runs at same time, they can be distinguished by a custom name.